index.js

```diff
@@ -11,5 +11,9 @@ module.exports = {
         return __dirname;
       }
     });
+  },
+
+  included() {
```

Comment:
We should generally super here

Reply:
Because of the node versions we marked as supported, I had to `this._super.apply(this, arguments)`.

Reply:
should be `this._super.included.apply(this, arguments)` (`this._super` essentially refers to our super prototype unbound)

Reply:
Is this different from the Ember object system `_super`? Or I'm just unfamiliar with what Ember does with `_super`?

Reply:
Correct, ember-cli uses `core-object` and it super implementation. It was intended as roughly an experiment for what _might_ work in ember itself, but native classes were introduced and that is clearly a better long term path for everyone.